src/command_modules/azure-cli-cosmosdb/HISTORY.rst

```diff
@@ -1,5 +1,10 @@
 .. :changelog:
 
+unreleased
+===============
+
+* Minor fix allowing Creation of Collection with custom partition key
```

Comment:
This should be under the `Release History` heading and should use `++++` instead of `=====`